R/LoadItaly.R

```diff
@@ -1,3 +1,16 @@
+#' dataQueryItaly
+#'
+#' @param date search for Italian COVID-19 data on this date
+#'
+#' @return returns data for Italy on specified date. Called by LoadItaly().
+#' @keywords internal
+dataQueryItaly <- function(date) {
+  denominazione_regione <- denominazione_provincia <- codice_provincia <- totale_casi <- NULL
+  data <- vroom::vroom(paste0("https://raw.githubusercontent.com/pcm-dpc/COVID-19/master/dati-province/dpc-covid19-ita-province-", stringr::str_replace_all(as.character(date), "-", ""), ".csv"), col_types = vroom::cols(note = vroom::col_character()), show_col_types = FALSE, progress = FALSE) %>%
+    dplyr::select(date = data, region = denominazione_regione, province = denominazione_provincia, code = codice_provincia, cases = totale_casi)
+  return(data)
+}
+
```

Comment:
Note that `LoadItaly` no longer calls `dataQueryItaly`. I changed the function to read the date from the ...latest.csv in the dataset, rather than trying to set it manually, given that data is no longer uploaded daily in this resource.

Reply:
Done!